src/pages/createCostModelWizard/filterToolbar.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  FormSelect,
+  FormSelectOption,
+  InputGroup,
+  InputGroupText,
+  TextInput,
+  ToolbarGroup,
+  ToolbarItem,
+} from '@patternfly/react-core';
+import { FilterIcon, SearchIcon } from '@patternfly/react-icons';
+import React from 'react';
+import { CostModelContext } from './context';
+import { addMultiValueQuery } from './filterLogic';
+
+const FilterToolbar = () => {
+  return (
+    <CostModelContext.Consumer>
+      {({
+        filterName,
+        onFilterChange,
+        page,
+        perPage,
+        query,
+        type,
+        fetchSources,
+      }) => {
+        return (
+          <ToolbarGroup>
+            <ToolbarItem>
+              <InputGroup>
+                <InputGroupText>
+                  <FilterIcon />
+                </InputGroupText>
+                <FormSelect aria-label="select filter type">
+                  <FormSelectOption value="name" label="Name" />
+                </FormSelect>
+              </InputGroup>
+            </ToolbarItem>
+            <ToolbarItem>
+              <InputGroup>
+                <TextInput
+                  value={filterName}
+                  placeholder="Filter by name"
+                  id="sources filter value"
+                  onChange={onFilterChange}
+                  onKeyPress={evt => {
+                    if (evt.key !== 'Enter' || filterName === '') {
+                      return;
+                    }
+                    fetchSources(
+                      type,
+                      addMultiValueQuery(query)('name', filterName),
+                      page,
+                      perPage
+                    );
+                  }}
+                />
+                <InputGroupText>
+                  <SearchIcon />
+                </InputGroupText>
```

Comment:
Similar issue as the filter icon. That is, the search icon appears outside the text field, surrounded by a border. The mockup shows something more like this:

<img width="345" alt="Screen Shot 2019-08-16 at 10 45 56 AM" src="https://user-images.githubusercontent.com/17481322/63175967-0c03d480-c013-11e9-984f-a0583e2f3c0d.png">